react-frontend-app/src/App.js

```diff
@@ -12,15 +12,18 @@ import './style/App.css';
 
 // Import different components
 import Description from './components/Description'
-import Display from './components/Display'
 import Footer from './components/Footer'
 import MainNavBar from './components/MainNavBar'
 import BarGraph from './components/BarGraph'
 
 function App() {
   const [surname, setSurname] = useState('');
   const [zipcode, setZip] = useState('');
-  var prediction = {};
+  const [loading, setLoading] = useState(true);
+
+  // var prediction = {};
```

Comment:
Could you add a comment here explaining what information is stored in "variable"? Right now, it's a little unclear